astro_2.0/features/pages/nestedDaoPagesContent/EditDraftPageContent/EditDraftPageContent.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { FC, useEffect, useState } from 'react';
+import isEmpty from 'lodash/isEmpty';
+
+import styles from 'pages/dao/[dao]/drafts/[draft]/edit-draft/EditDraftPage.module.scss';
+
+import { BackButton } from 'astro_2.0/features/ViewProposal/components/BackButton';
+import { CreateProposal } from 'astro_2.0/features/CreateProposal';
+
+import { DRAFTS_PAGE_URL } from 'constants/routing';
+import { getInitialFormValuesFromDraft } from 'astro_2.0/features/ViewProposal/helpers';
+
+import { useDaoCustomTokens } from 'context/DaoTokensContext';
+import { useWalletContext } from 'context/WalletContext';
+
+import { DaoContext } from 'types/context';
+import { DraftProposal } from 'types/draftProposal';
+
+interface Props {
+  daoContext: DaoContext;
+  draft: DraftProposal;
+}
+
+export const EditDraftPageContent: FC<Props> = ({ daoContext, draft }) => {
+  const { dao } = daoContext;
+  const [initialValues, setInitialValues] = useState({});
+  const { tokens } = useDaoCustomTokens();
+  const { accountId } = useWalletContext();
+
+  useEffect(() => {
+    const getInitialValues = async () => {
+      if (tokens) {
+        const createdInitialValues = await getInitialFormValuesFromDraft(
+          draft?.proposalVariant,
+          (draft as unknown) as Record<string, unknown>,
+          tokens,
+          accountId || ''
+        );
+
+        setInitialValues({
+          ...createdInitialValues,
+          id: draft.id,
+          state: draft.state,
+          externalUrl: '',
+          title: draft.title,
+          description: draft.description,
+          details: draft.description,
+        });
+      }
+    };
+
+    getInitialValues();
+  }, [accountId, draft, tokens]);
+
+  return (
+    <div className={styles.draftInfo}>
+      <BackButton
+        name="Back to Draft Feed"
+        href={{
+          pathname: DRAFTS_PAGE_URL,
+          query: {
+            dao: dao.id,
+          },
+        }}
+      />
+      {!isEmpty(initialValues) ? (
```

Comment:
I consider this can be shortened to 
`
{!isEmpty(initialValues) && <CreateProposal
          proposalVariant={draft.proposalVariant}
          showInfo={false}
          showClose={false}
          showFlag={false}
          dao={dao}
          daoTokens={tokens}
          onClose={() => undefined}
          initialValues={initialValues}
          userPermissions={daoContext.userPermissions}
          isDraft
          isEditDraft
        />}
`